lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -140,7 +142,7 @@ describe('PerfCategoryRenderer', () => {
     const diagnosticSection = categoryDOM.querySelectorAll('.lh-category > .lh-audit-group')[2];
 
     const diagnosticAudits = category.auditRefs.filter(audit => audit.group === 'diagnostics' &&
-        audit.result.score !== 1 && audit.result.scoreDisplayMode !== 'not-applicable');
+        audit.result.score < PASS_THRESHOLD && audit.result.scoreDisplayMode !== 'not-applicable');
```

Comment:
can just use `Util.PASS_THRESHOLD` instead of the local declaration above. 

i think that'll be fine despite all the jsdom beforeEach bs. but holler if its stupid.